op-guide/binary-deployment.md

```diff
@@ -1,28 +1,164 @@
+
 ---
 title: TiDB Binary 部署方案详解
 category: deployment
 ---
 
-# TiDB Binary 部署方案
+# TiDB Binary 部署指导
+
+
+目录
+=================
```

Comment:
better do not add TOC.

``====`` for a title conflicts with all current scripts.

Reply:
remove TOC